### DISCUSSION TITLE
ID.4: stop charging (by unlocking twice?)

### DISCUSSION THREAD

**Discussion by mivola** [tillsteinbach/WeConnect-mqtt#354] [General]
Hi,

I just found this project, got it running in a Docker container and now receive a lot of data about my car. So first of all thanks a lot to @tillsteinbach for this great project!

Now I wonder if there is a way to stop the car charging (AC) - typically I do this by unlocking the car (twice) via the remote control. Do you think this would be possible to do via the MQTT writable topics?

Thanks a lot,
Michael

**Comment by tillsteinbach** [comment]
Yes send 'stop' to topic '/vehicles/[YOUR VIN]/controls/charging_writetopic'

**Reply by mivola** [reply]
> Yes send 'stop' to topic '/vehicles/[YOUR VIN]/controls/charging_writetopic'

That works perfectly! Thx a lot!
